examples/js/loaders/GLTFLoader.js

```diff
@@ -1088,8 +1088,14 @@ THREE.GLTFLoader = ( function () {
 
 					materialParams.emissive = new THREE.Color().fromArray( materialValues.emission );
 
+				} else if ( typeof( materialValues.emission ) === 'string' ) {
+
+					materialParams.map = dependencies.textures[ materialValues.emission ];
+
 				}
 
+				delete materialParams.emission;
```

Comment:
Ok, I've reworked this a little, only leaving `delete` where it's actually necessary (for `specular`).